simulation-runner/config/scenarios/foss-puppetserver210-catalogzero-1.json

```diff
@@ -0,0 +1,13 @@
+{
+    "run_description": "puppeserver 2.1.0 Catalog Zero - 1000 agents, 30 minute ramp, 2 times",
```

Comment:
Looks like the other scenarios are keeping the descriptions up to date with the configurations...we might as well correct this one too.  

Something like `1 agent, 2 times`?